common/actions/deterministicWallets.js

```diff
@@ -78,7 +78,7 @@ export function setDesiredToken(token: ?string): SetDesiredTokenAction {
 export type UpdateDeterministicWalletArgs = {
   address: string,
   value: ?Big,
-  tokenValues: ?TokenValues
+  tokenValues: TokenValues
```

Comment:
What was complaining about this? It was expected that sometimes you'd only want to update value, or only tokenValues.